cms/templates/base.html

```diff
@@ -85,7 +85,7 @@
                       url_name: "${context_course.location.name | h}",
                       org: "${context_course.location.org | h}",
                       num: "${context_course.location.course | h}",
-                      display_course_number: "${_(context_course.display_number_with_default)}",
+                      display_course_number: "${_(context_course.display_coursenumber)}",
```

Comment:
Are we sure this will address the issue?  When I look at the implementation for the 'display_number_with_default' property it attempts to return 'display_coursenumber'.  If there is no value available for 'display_coursenumber' the property will return the value for 'number'.  So it seems something else may be going on here.  Is it possible to include a test configuration which exposes the exception state and shows we've corrected the problem?